docs/src/user-guide/quick-start-cluster-setup/single-node.md

```diff
@@ -7,6 +7,7 @@ A single-node deployment allows you to run CLP on a single host.
 * [Docker]
   * If you're not running as root, ensure `docker` can be run
     [without superuser privileges][docker-non-root].
+  * If you're using Docker Desktop, you will need version 4.34 or later, and [enable host networking][docker-desktop-host-networking].
```

Comment:
let me know if we want to also update the multi-node cluster requirements section. i intentionally skipped the update there because launching a multi-node cluster with Docker Desktop might be an uncommon action, unless for trial / experimental purposes.